src/services/user-guest-service.ts

```diff
@@ -1,46 +1,53 @@
-import {UserGuest} from '../models/user-guest'
+import moment from 'moment'
+import { UserGuest } from '../models/user-guest'
 import { v4 as uuidv4 } from 'uuid'
 
 export class UserGuestService {
-    private static USER_GUEST_KEY = 'userGuest'
-
-    public static create(firstName: string, lastName: string, dateOfBirth: Date, typeOfIdentityDocument: string, identityDocumentNumber: string) {
-        return {
-            id: uuidv4(),
-            firstName: firstName,
-            lastName: lastName,
-            dateOfBirth: dateOfBirth.getTime(),
-            typeOfIdentityDocument: typeOfIdentityDocument,
-            identityDocumentNumber: identityDocumentNumber
-        }
+  private static USER_GUEST_KEY = 'userGuest'
+
+  public static create(
+    firstName: string,
+    lastName: string,
+    dateOfBirth: Date,
+    typeOfIdentityDocument: string,
+    identityDocumentNumber: string
+  ) {
+    return {
+      id: uuidv4(),
+      firstName: firstName,
+      lastName: lastName,
+      dateOfBirth: moment(dateOfBirth).format('DD-MM-YYYY'),
```

Comment:
Hay un método en `dateHelper.ts`que justo hace eso :D 

```suggestion
            dateOfBirth: formatDate(dateOfBirth),
```